fvm/src/gas/price_list.rs

```diff
@@ -18,100 +18,106 @@ use num_traits::Zero;
 
 use super::GasCharge;
 
+macro_rules! to_milligas {
+    ($ex:expr) => {
+        $ex * crate::gas::MILLIGAS_PRECISION
+    };
+}
+
 lazy_static! {
     static ref OH_SNAP_PRICES: PriceList = PriceList {
-        compute_gas_multiplier: 1,
-        storage_gas_multiplier: 1300,
+        compute_gas_multiplier: to_milligas!(1),
```

Comment:
If `compute_gas_multiplier` was used, this would be wrong (you would end up multiplying by 1000 twice) but it looks like `compute_gas_multiplier` is not used.

Reply:
We should probably remove it from the price list if it isn't going to be used.

Reply:
Ack.

Reply:
Yes, this, and the storage gas multiplier were wrongly scaled to millis.